src/crewai/agents/executor.py

```diff
@@ -185,6 +198,55 @@ def _iter_next_step(
             yield AgentStep(action=output, observation=observation)
             return
 
+        except Exception as e:
```

Comment:
_Question_: Isn't possible to raise the specific type of Exception that is raised when the contextExceeded limit is raised?

Reply:
not Explicitly the ContextWindowError also we need to ensure this gets captured for all the other model providers - groq, cohere, and anthropic